astro/src/content/blog/test-saml-configuration.mdx

```diff
@@ -0,0 +1,92 @@
+---
+publish_date: 2025-01-27
+title: "Test Your FusionAuth SAML Configuration"
+description: 'How to test your FusionAuth SAML configuration.'
+authors: Mark Robustelli
+image: /img/blogs/test-saml-configuration/test-saml-configuration-header.png
+categories: Tutorial
+tags: saml, configuration
+excerpt_separator: "{/* more */}"
+---
+
+import Aside from '/src/components/Aside.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+Setting up authentication for your application can feel a bit like poking around in the dark sometimes. So many nerd knobs to tweak. You have read the documentation, triple checked your settings, but you're still not sure if you have everything dialed in. Good news, there is a way to test and make sure your SAML v2 configuration is set up correctly. Enter [IAMShowcase](https://sptest.iamshowcase.com/). [IAMShowcase](https://sptest.iamshowcase.com/) is a site that acts as a SAML v2 SP (service provider) to test your FusionAuth IdP (Identity Provider). The authentication world is full of acronyms and that last sentence is no exception. So, grab a cup of coffee (or something stronger if you're so inclined) and let's break it down.
+
+{/* more */}
+## Acronyms and Jargon and Bears, Oh My!
+If you are familiar with acronyms like SAML, SP, and IdP feel free to skip to the next section. For those of us that are merely mortal, here are some definitions.
+
+**[SAML v2](/docs/lifecycle/authenticate-users/saml)** - Security Assertion Markup Language - SAML is like a digital passport. It lets users log into one service and also gain access to another service without needing to login again. It is a standard way to verify a user's identity and authorize access to applications and services. The user can log in once and access multiple applications.
+
+**SP** - Service Provider - Think of this as an application or service your user wants to access. In this case IAMShowcase will be the SP.
+
+**IdP** - Identity Provider - A service that verifies your user's identity. In this case, FusionAuth. This will be where the user logs in. 
```

Comment:
```suggestion
**IdP** is short for "Identity Provider". An IdP is a service that verifies your user's identity and communicate that identity in a secure way to any applications that depend on it. This is where the user logs in. FusionAuth is the IdP for this tutorial. 
```

Reply:
I would move this to ahead of the SP definition so you can use the IdP term in the SP definition.